File: sdk/java-sdk/src/main/java/com/evernym/verity/sdk/exceptions/WalletClosedException.java
package com.evernym.verity.sdk.exceptions;

/**
 * Exception class that is thrown when the wallet is un-expectantly closed
 */
public class WalletClosedException extends WalletException {
    /**
     * Constructs a new WalletClosedException
     *
     * @see VerityException#VerityException(String)
     */
    public WalletClosedException() {
        super("A context wallet handle accessed after being closed");
    }
}

File: samples/sdk/dotnet-example-app-tmp/VeritySDK/Exceptions/WalletClosedException.cs
namespace VeritySDK.Exceptions
{
    ///// <summary>
    ///// Exception class that is thrown when the wallet fails to open
    ///// </summary>
    public class WalletClosedException : WalletException
    {
        /// <summary>
        /// Constructs a new exception.
        /// </summary>
        public WalletClosedException() : base("A context wallet handle accessed after being closed") { }
    }
}
